packages/theme-language-server-common/src/server/startServer.ts

```diff
@@ -387,8 +385,10 @@ export function startServer(
   });
   connection.workspace.onDidRenameFiles((params) => {
     const triggerUris = params.files.map((fileRename) => fileRename.newUri);
+    renameHandler.onDidRenameFiles(params);
```

Comment:
Stuck while tophatting:
- I have using "Debug node extension"
- When i rename a css or js file, i never see a `workspace/didRenameFiles` in the logs
- I placed a breakpoint here just to make sure, but never hit - it does work for liquid + json files

Reply:
23f980e1ae52210546d620adeea2a6c7053f1025